flipperdevices/Flipper-Android-App
# Open tools tab one time after start

**Background**

To show users new infrared-remotes feature we should navigate them somehow into tools tab. This pr changes some proto settings so Tools tab will be the first after app is intalled/open after update

**Changes**

- Change proto files to open tools tab first

**Test plan**

- Open non-updated app on not-tools tab
- Update up
- Open app and see you are on tools tab
- Reinstall app, and see the first tab is now tools
